cryptol-saw-core/src/CryptolSAWCore/Simpset.hs

```diff
@@ -21,13 +21,14 @@ import SAWCore.SharedTerm
 mkCryptolSimpset :: SharedContext -> IO (Simpset a)
 mkCryptolSimpset sc =
   do m <- scFindModule sc cryptolModuleName
-     scSimpset sc (cryptolDefs m) [] []
+     scSimpset sc (cryptolDefs m) idents []
   where
     cryptolDefs m = filter (not . excluded) $ moduleDefs m
     excluded d =
       case nameInfo (defName d) of
         ModuleIdentifier ident -> ident `elem` excludedNames
         ImportedName{} -> True
+    idents = ["Prelude.coerce_same", "Prelude.unsafeCoerce_same"]
```

Comment:
I know that `cryptol_ss` is already a bit ad hoc, but it is interesting that we need to specifically add identifiers from the SAWCore prelude, not `cryptol-saw-core`'s prelude, in order to make things work. This raises a number of questions when I read it:

* What determines which identifiers to add to `cryptol_ss`? The SAW user manual entry gives the impression that it contains primarily Cryptol-specific rules, but clearly there's more to it than that.
* Would these be better suited for `basic_ss` than `cryptol_ss`? I'm not actually sure if `cryptol_ss` is a superset of `basic_ss`, however.
* Would it be worth documenting in more detail what `cryptol_ss`/`basic_ss` contain in the SAW user manual?